Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.yml

```
@@ -15,6 +15,8 @@ args:
 - description: Public IP address of the EC2 instance.
   name: public_ip
   required: true
+- description: If assuming roles for AWS, this is the name of the role to assume (should be the same for all organizations).
```

Wouldn't something like

```suggestion
- description: Name of an AWS role to assume (should be the same for all organizations).
```
be more concise and in-line with the other descriptions?

(If you do change that, make sure to also change in the README)

---

i don't think I want to change because I want to note that it is actually optional

---

We don't specify in the descriptions that parameters that they are optional (unless there's some specific reason it's needed), we use the `required` field for that.

You can check descriptions for optional parameters in commands to see examples.